packages/glob/src/glob.ts

```diff
@@ -1,183 +1,27 @@
 import * as core from '@actions/core'
-import * as fs from 'fs'
-import * as path from 'path'
-import * as patternHelper from './internal-pattern-helper'
-import {IGlobOptions} from './internal-glob-options'
-import {MatchKind} from './internal-match-kind'
-import {Pattern} from './internal-pattern'
-import {SearchState} from './internal-search-state'
+import {Globber, DefaultGlobber} from './internal-globber'
+import {GlobOptions} from './internal-glob-options'
 
-export {IGlobOptions}
+export {Globber, GlobOptions}
 
 /**
- * Returns files and directories matching the specified glob pattern.
+ * Contructs a globber from an input
  *
- * Order of the results is not guaranteed.
+ * @param name  Name of the input
  */
-export async function glob(
-  pattern: string,
-  options?: IGlobOptions
-): Promise<string[]> {
-  const result: string[] = []
-  for await (const itemPath of globGenerator(pattern, options)) {
-    result.push(itemPath)
-  }
-  return result
+export async function getInput(
```

Comment:
now this file is only used to export the interfaces (Globber, GlobOptions) and instantiate a globber (getInput, parse)

everything else moved into internal-globber.ts